File: src/main/java/com/flab/license/api/HealthController.java
package com.flab.license.api;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import com.flab.license.common.response.ApiResponse;

@RestController
public class HealthController {

	@GetMapping("/health")
	public ApiResponse<Void> health() {
		return ApiResponse.ok();
	}
}

File: src/main/kotlin/com/flab/license/api/HealthController.kt
package com.flab.license.api

import com.flab.license.common.response.ApiResponse
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HealthController {

    @GetMapping("/health")
    fun health(): ApiResponse<Unit> = ApiResponse.ok()
}
